_mental_models/saying-no.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: saying-no
+name: Saying "No"
+benefit: Better Decisions
+summary: Saying "yes" to requests because you don't want to upset people can cause problems.
+prerequisites: Confidence, certainty
+---
+
+#### Description
+
+I say "yes", because I don't want to be rude to people and mostly when I say "no", I feel that person feels bad about it. There might be different cases for saying "yes", but in my case, I was thinking that saying yes to an offer, question or a request makes the thing easier.
+
+But saying some requests "yes" can cause some problems, like burn-out or feeling bad. In some cases you might be ended up with saying "yes" to a case where can't handle.
+
+So being able to say "no" to certain things can help to maintain your mental health and can provide better decision making in long-run.
+
+#### Practice
+
+1. Make sure you are relaxed enough and you are not distracting by some emotions like anger.
+2. Find some cases from your previous experiences that you ended up saying "yes" and write down the cases what could have been if you say "no" to these questions. (The questions that caused some problems, frustration or that you felt bad)
+3. Understand everyone can have that kind of problems and saying "no" to things is not the end of the world. Thinking about that in your relaxed state can help you to understand better. In emotional state you can't understand it well.
+
+In case you should say "no", think about these :
+
+1. Be concise and express your feelings without feeling guilty
+2. People can understand you if you be nice and reasonable to them. If you explain why you say "no", that might change whole situation. And that can also helps you to maintain your relationship with this person.
+3. It's okay to say "I need some space" or "I'm unable to participate", there is nothing wrong about that.
+
```

Comment:
#### Practice 2 - [Simulation](https://mmpractices.com/mental_models/simulation/)

1. Imagine someone requesting something from you.
2. Imagine replying back to them saying "no"

Note that all of these examples & practices are to develop your "no" muscle. You should go as wild as you can with the simulations.

##### Example 1

1. I imagine myself going down the street in a chill mode. Some old lady requests me to help with carrying her bags.
2. I imagine saying "no I am in a hurry", doesn't matter if I am or I am not. It will train my "no" muscle.

##### Example 2

1. I imagine being on my computer chilling and my friend texts me that I need to fix/do something.
2. I imagine saying "no, I am exhausted today, I'm in my chill time, I'll look at it tomorrow"

##### Example 3

1. I imagine my sister wanting me to get up and check the food in 10 minutes.
2. I imagine telling her "no, I am concentrated right now. You need to figure out another method which doesn't involve me"